code/bit-manipulation/Unique_no_2.cpp

```diff
@@ -0,0 +1,45 @@
+void findUnique2(int *a,int n){
```

Comment:
Add all of your includes for this file, otherwise it won't compile. Don't include `<bits/stdc++.h>`. See #1915.